packages/SwingSet/src/kernel/virtualObjectManager.js

```diff
@@ -468,12 +623,37 @@ export function makeVirtualObjectManager(
         if (vkey) {
           assert(syscall.vatstoreGet(vkey), X`${q(keyName)} not found: ${key}`);
           syscall.vatstoreDelete(vkey);
+          removeRecognizableValue(key, deleter);
```

Comment:
let's add a TODO here: decrement refcounts on vrefs in data, to remind us that this is where the "down" side of refcounting needs to be placed

Reply:
No point.  The next PR in line already does the stuff that the comment would remind us to do.